dev/tools/localization/gen_l10n.dart

```diff
@@ -515,22 +525,40 @@ class LocalizationsGenerator {
     return !(statString[1] == 'w' || statString[4] == 'w' || statString[7] == 'w');
   }
 
-  /// Sets the reference [Directory] for [l10nDirectory].
+  /// Sets the reference [Directory] for [inputDirectory].
   @visibleForTesting
-  void setL10nDirectory(String arbPathString) {
-    if (arbPathString == null)
-      throw L10nException('arbPathString argument cannot be null');
-    l10nDirectory = _fs.directory(arbPathString);
-    if (!l10nDirectory.existsSync())
+  void setInputDirectory(String inputPathString) {
+    if (inputPathString == null)
+      throw L10nException('inputPathString argument cannot be null');
+    inputDirectory = _fs.directory(inputPathString);
+    if (!inputDirectory.existsSync())
       throw FileSystemException(
-        "The 'arb-dir' directory, $l10nDirectory, does not exist.\n"
+        "The 'input-dir' directory, $inputDirectory, does not exist.\n"
```

Comment:
When printing file names, it's probably best to surround them with quotes, in case they contain spaces or special characters.